### PR TITLE
Fix LOD dithering artifacts

### LINKED ISSUE #1388: White pixel flickering artifacts with "DitherFade" material layer
This may be related to #1104, but that issue is about seeing temporal dithering artifacts with forward rendering + AA. There are actually dithering artifacts that happen _with_ temporal rendering.

Reported on the forum [here](https://community.cesium.com/t/light-white-pixel-flickering-on-tiles/31203/). The user sees bright white pixels flickering throughout the tileset, which are especially visible when the lighting is dark. This is with:

- Unreal Engine 5.3
- Cesium for Unreal 2.4.1
- Google Photorealistic 3D Tiles
- Temporal Super Resolution enabled in project settings (this is how it's set in the samples)
- **Enable LOD Transitions** on the tileset set to false

To reproduce the view, set the georeference to
**Latitude**: 59.908135
**Longitude**: 10.721472
**Height**: 99.238093

image
( Zoom in to see more clearly. From a distance it will look like you have dust on your computer. :') )

The workaround was to delete the "DitherFade" material layer from the default tileset material. 

One action item is to test if the plugin version matters, since the user says it worsened with time. Engine version may matter too.

### PR DESCRIPTION
**Pull request by j9liu**
Fixes #1388, #1359 

This PR modifies the ML_DitherFade material layer to prevent some flickering pixels that could be seen on tilesets. These were most visible on Google Photorealistic 3D Tiles, and especially when the tileset was selected. You can see it in `12_CesiumGoogleMapsTiles` behind the intro billboard, or check out #1388 for an example.

### Before

image

image

### After

image

image

I will also test if this addresses #1104, TBD.

### CODE REVIEW

File: Source/CesiumRuntime/Private/CesiumGltfComponent.cpp [hunk 1]
```diff
@@ -3582,10 +3580,12 @@ void UCesiumGltfComponent::UpdateFade(float fadePercentage, bool fadingIn) {
     return;
   }
 
+  fadePercentage = glm::clamp(fadePercentage, 0.0f, 1.0f);
```
Comment (csciguy8) [inline Source/CesiumRuntime/Private/CesiumGltfComponent.cpp:3583]: Did moving this line fix anything? Or was it just code cleanup?

Comment (j9liu) [reply Source/CesiumRuntime/Private/CesiumGltfComponent.cpp:3583]: Just code cleanup from when I was debugging the fades (which was ultimately not necessary)

**Comment by j9liu** [comment]
Unfortunately this does not address #1104. ☹️

**Comment by kring** [comment]
Code looks good, and this is working well, no matter if LOD transitions are enabled or disabled. Merging.

**Pull request merged**